sonar-plugin-api/src/main/java/org/sonar/api/ce/posttask/PostProjectAnalysisTask.java

```diff
@@ -92,10 +92,18 @@
      * This date is the same as the date of the project analysis report and therefore as the analysis in DB. It can be
      * missing when the status of the task is {@link org.sonar.api.ce.posttask.CeTask.Status#FAILED FAILED}.
      * </p>
+     * @deprecated use {@link #getAnalysis().getDate()} instead
      */
+    @Deprecated
     Optional<Date> getAnalysisDate();
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)